login-module/src/main/java/com/wipro/ats/bdre/security/UserRoleFetcher.java

```diff
@@ -100,9 +94,8 @@ public Users getUser(String username) {
                 userRoleList.add(userRole);
 
             }
-            // userroles = s.selectList("call_procedures.ListUserRoles", userrole);
 
-        } catch (Exception e) {
+        } catch (BDREException e) {
             throw new RuntimeException(e);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00112)